### PR TITLE
[runtime] use fst instead of far

### PR DESCRIPTION
**Pull request by pengzhendong**

### CODE REVIEW

File: runtime/processor/token_parser.cc [hunk 1]
```diff
@@ -148,7 +148,6 @@ std::string TokenParser::reorder(const std::string& input) {
   for (auto& token : tokens) {
     output += token.string(orders) + " ";
   }
-  tokens.clear();
```
Comment (xingchensong) [inline runtime/processor/token_parser.cc:151]: Why delete tokens.clear() ?

Comment (pengzhendong) [reply runtime/processor/token_parser.cc:151]: Cause `token_parser` is a local var now.

File: runtime/processor/processor.h [hunk 2]
```diff
@@ -15,31 +15,31 @@
...
   std::string normalize(const std::string& input);
 
  private:
-  std::string compose(const std::string& input, const Fst<StdArc>* fst);
+  std::string compose(const std::string& input, const StdVectorFst* fst);
 
   ParseType parse_type_;
-  Fst<StdArc>* tagger_ = nullptr;
-  Fst<StdArc>* verbalizer_ = nullptr;
-  std::shared_ptr<StringCompiler<StdArc>> compiler_ = nullptr;
```
Comment (xingchensong) [inline runtime/processor/processor.h:42]: I prefer `shared_ptr`,  when integrating into other projects (wenet wetts), we can shared resources

File: runtime/processor/token_parser.cc (hunk 1 above)
Comment (xingchensong) [reply runtime/processor/token_parser.cc:151]: okay

**Review by xingchensong — APPROVED** [review]

**Pull request merged**